README.md

```diff
@@ -19,8 +19,8 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
-* [Website](https://nodejs.org/en/)
-* [Contributing to the project](./CONTRIBUTING.md)
+* Website: [nodejs.org](https://nodejs.org/en/)
+* Project Contributing: [CONTRIBUTING.md](./CONTRIBUTING.md)
```

Comment:
I am not a native English speaker, but «Project Contributing» does sound a little suspicious to me.
/cc @nodejs/documentation